### ISSUE TITLE
Panasonic Remote Camera Controller RP150 random reboot when management Companion

### ISSUE THREAD

**Issue by TRT-777** [bitfocus/companion-module-panasonic-camera-controller#30]
When quickly switching (1-2 sec) cameras, only from different groups, using companion, the RP150 randomly reboots. 
We have 4 RP150 remote controls and they all reboot. 
We tried changing the firmware, power supply, it does not work!
Does anyone have the same problem or not? If after selection you wait until the remote control receives a response from all cameras in the group, then this problem does not exist! But this is extra time (about 5-12sec), which is sometimes not there. Control is carried out by 30-40 Panasonic PTZ cameras of the 80, 150 series. Control is carried out via LAN. Unconnected cameras in the remote control are disabled. It is necessary that the remote control does not reboot.
We use companion because switching from RP150 cameras is not convenient and we use companion that controls multiple devices at the same time.

**Comment by TRT-777** [comment]
I don't think this is a Companion bug. Maybe someone tried to find a solution to this problem

**Comment by Seanbucks** [comment]
You're not alone, this is a real issue and most apparent when jumping between groups. I was able to narrow it down recently and help with a bug fix, but it's going to be a bit longer before there's an update and unfortunately in the meanwhile, your options are limited. You can dedicate one RP-150 controller per group to avoid group changes and crashes. Use the RP-60 instead, though smaller groups and slower than it should be on group change, it is far less likely to crash. You can also convert control over to RS422 and use a serial data router, but it's complicated and costly.

**Comment by TRT-777** [comment]
Ok. Thank you!
